notifications/management/commands/populate_notification_settings.py

```diff
@@ -12,15 +12,10 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument("--username", nargs="?", action="append")
-        parser.add_argument("--inactive", action="store_true", default=False)
 
     def handle(self, *args, **options):
         users = get_user_model().objects.all()
 
-        if not options["inactive"]:
-            # unless the --inactive flag is specified, default to the safer subset of active users
-            users = users.filter(profile__last_active_on__isnull=False)
```

Comment:
Instead of deleting this option, maybe change to this?:

```
users = users.filter(last_login__isnull=False)
```

Reply:
I'm not sure why this option is needed. @rhysyngsun you added it a long time ago, do you remember why it exists and is it ok to get rid of it?

The Notification Settings page assumes that all users have comment and front page notification settings and breaks if they don't have them, so we don't want users without notification settings. Also the job skips over users with existing notification settings and makes no changes.

Reply:
That can be removed too, it's a holdover from the massive micromasters backpopulate